packages/core/src/util/renderer.ts

```diff
@@ -98,6 +99,20 @@ export const computeLabel = (
   return required && !hideRequiredAsterisk ? label + '*' : label;
 };
 
+/**
+ * Indicates whether to mark a field as required.
+ *
+ * @param {boolean} required whether the label belongs to a control which is required
+ * @param {boolean} applied hideRequiredAsterisk option
```

Comment:
The first word should be the parameter name as it's specially handled by the JSDoc
```suggestion
 * @param {boolean} hideRequiredAsterisk applied UI Schema option
```

Reply:
Thank you for the hint, i updated the doc